module/os/linux/zfs/zfs_znode.c

```diff
@@ -1111,6 +1111,10 @@ zfs_zget(zfsvfs_t *zfsvfs, uint64_t obj_num, znode_t **zpp)
 			mutex_exit(&zp->z_lock);
 			sa_buf_rele(db, NULL);
 			zfs_znode_hold_exit(zfsvfs, zh);
+			/* if znode is already marked for deletion, break out early */
```

Comment:
This line throws a style tantrum, might want to rephrase or split

Reply:
Oops, thanks for the notice. I didn't realize tabs count as 8 spaces, but it should be addressed now.

Reply:
I didn't know either, maybe rewrite the test a bit someday... Because this should've fitted fine.
So not really your fault either :)